05/Validation.js

```diff
@@ -0,0 +1,17 @@
+import fields from './fields';
```

Comment:
Tutaj podobnie, nie jest to konstruktor ani komponent więc nazwa pliku z małej litery :)